mailpoet/RoboFile.php

```diff
@@ -43,6 +49,119 @@ public function update() {
       ->run();
   }
 
+  public function updateJsPackages($opts = ['ticket' => '', 'latest' => true, 'checks' => false, 'dedupe' => false]) {
+    $ticket = $opts['ticket'];
+    $latest = $opts['latest'];
+    $runChecks = $opts['checks'];
+    $runDedupe = $opts['dedupe'];
+
+    if (empty($ticket)) {
+      $this->say('Please specify a ticket with --ticket=<ticket>');
+      exit(1);
+    }
+
+    $outdatedPackagesOutput = $this->taskExec('pnpm outdated --no-table')
+      ->printOutput(false)
+      ->run()
+      ->getMessage();
+    $lines = explode("\n", $outdatedPackagesOutput);
+
+    // The package names themselves are every third line
```

Comment:
Maybe the format was meant to be split on `\n\n`? That said, your approach seems to work well...